server-code/webhook.pl

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/perl
+# !!!REMINDER!!!: I have removed the secret key, you must
+# put this in line 26 for the script to function.
```

Comment:
Couple questions so I understand:

- does this also deal with merges in to master on PR completion or just tagged versions?
- Do we want/need to log out all the info from every webhook request? Maybe it's fine I just don't know how large that file might grow with a lot of noise that's not useful. Not a big deal but something to keep an eye on.

Reply:
At first, I was going to do somethings on PR completion, so there is code that listens for it.  In the end I decided to just respond to tag creation, since we need to do the tag creation anyway and that is the only way to know the appropriate tag name to apply.
The log info gets rewritten on every request, so no worry about file size growth.  There is not a whole lot of point to the log anyway, it just shows you what the last request to be processed was.  Maybe I will beef up the logging in the future, but let's give a few weeks of testing first.